### DISCUSSION TITLE
HDRP 사용가능한가요?

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1268] [5. Q&A]
URP는 사용가능하다고 들었는데, HDRP는 사용할 만 할까요?
Material이 전부 Pink로 변하는 게 문제인 거 같긴 해요..
근데 유튜브에 보니까 거의다 Shader는 Urp아니면 HDRP로 만드는 튜토리얼이 많아서
쓸 수 있으면 좋을텐데 혹시 이와 관련해서 아시는 분 계신가요?

HDRP가 지원되지 않는다면 쉐이더 어떻게 작성하시나요?

**Comment by YSHFM** [comment]
https://www.mettit.com/challenge/applcn-view/CPN20220816160051228

해당 링크 보시다시피 정상적으로 쉐이더 스크립트가 제페토 내부적으로 동작합니다.

하지만 HDRP의 경우, 제페토앱 내부적으로 LDR로 렌더링 되도록 설정이 고정되어 있는 것으로 보여서 HDR이 동작되진 않았습니다.
따라서 HDRP를 적용하더라도 의도된 아트 디자인이 아닐 수 있다고 생각됩니다.

또한, https://studio.zepeto.me/kr/guides/unity_package_list 해당 제페토 패키지 지원 목록을 보면 HDRP패키지가 지원 되지 않는 것으로 보여서 프로젝트에 HDRP패키지를 적용하더라도 빌드에 포함되지 않을 것 같습니다.

**Comment by chlguswls32** [comment]
안녕하세요. Sonak0930님.
제페토에서 현재 URP(Universal Render PipeLine)을 지원안하는것으로 알고있습니다. 

따라서 쉐이더 작성은 Shader Graph에서 불가능하고, ShaderCode를 통해서 작성하여야 할것같습니다.
